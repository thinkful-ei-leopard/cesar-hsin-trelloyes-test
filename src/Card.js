import './Card.css';
import React from 'react';

function Card(props){
    console.log(props);
    return(
        <div className="Card" key={props.id}>
            <button type="button">delete</button>
            <h3>{props.title}</h3>
            <p>{props.content}</p>
          </div>
    )
}

export default Card;