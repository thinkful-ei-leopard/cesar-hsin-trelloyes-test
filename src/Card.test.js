
/*smoke test
import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';


 describe ('<Card />', () =>{
    it ('renders without crashing' , () => {
        const div = document.createElement('div');
    
    ReactDOM.render(
        <Card  
        title="Example title" 
        content="Example content"
        id={10} />, 
        div);
        ReactDOM.unmountComponentAtNode(div);
    })

})*/

//Snapshot test

import React from 'react';
import  ReactDOM from 'react-dom';

import Card from './Card';

describe('<Card />', ()=>{
  it('render this as expected', ()=> {
      const test = document.createElement('div');

      ReactDOM.render(<Card />, test);
      ReactDOM.unmountComponentAtNode(test);

  });
});
