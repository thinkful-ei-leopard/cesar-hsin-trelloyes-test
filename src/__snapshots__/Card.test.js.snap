// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render this as expected 1`] = `<div />`;
